### ISSUE TITLE
[1.7.0rc2] Incorrect URL for 'Export' button in Plugin Manager when OctoPrint is not served under the server root

### ISSUE THREAD

**Issue by The-EG** [OctoPrint/OctoPrint#4231]
### The problem

Whenever OctoPrint is being served under a folder, not the root of the server, ie. `http://(ip)/ender-3-v2/`, the 'Export' button at the top of the Plugins Manager screen does not prefix the correct path.

Instead of linking to `http://(ip)/ender-3-v2/plugin/pluginmanager/export` the button instead targets `http://(ip)/plugin/pluginmanager/export`. If I manually go to `http://(ip)/ender-3-v2/plugin/pluginmanager/export` I get the expected file.

![image](https://user-images.githubusercontent.com/7342077/132961008-c4611370-c21a-4f59-a21f-3b537e53f87e.png)

### Did the issue persist even in safe mode?

Yes, it did persist

### Version of OctoPrint

1.7.0rc2

### Operating system running OctoPrint

Raspberry Pi OS (buster)

### Browser and version of browser, operating system running browser

Waterfox G3.2.4.1

### Checklist of files to include below

- [X] Systeminfo Bundle (always include!)
- [ ] Contents of the JavaScript browser console (always include in cases of issues with the user interface)
- [X] Screenshots and/or videos showing the problem (always include in case of issues with the user interface)
- [ ] GCODE file with which to reproduce (always include in case of issues with GCODE analysis or printing behaviour)

### Additional information & file uploads

[octoprint-systeminfo-20210911163943.zip](https://github.com/OctoPrint/OctoPrint/files/7148492/octoprint-systeminfo-20210911163943.zip)

I'm using nginx as the proxy with the following configuration:
```
server {
       listen 80;
       listen [::]:80;

       root /opt/printerweb;

       index index.html index.htm;

       server_name _;

       location /ender-3-v2/ {
                proxy_pass http://localhost:5001/;
                proxy_set_header Host $http_host;
                proxy_set_header Upgrade $http_upgrade;
                proxy_set_header Connection "upgrade";
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header X-Scheme $scheme;
                proxy_set_header X-Script_name /ender-3-v2;
                proxy_http_version 1.1;

                client_max_body_size 0;
       }
}
```

**Comment by cp2004** [comment]
Hmm, that shouldn't have been overlooked... I believe I added that button to the UI, so I'll look at sorting it

**Comment by foosel** [comment]
@cp2004 I hope you don't mind that I quickly fixed this. I hope to be able to run through the update tests today and ideally also push out the new RC and that was kinda blocking it 😅

Trivial fix really, and overlooked during review, so this was my mistake as well 😊

**Comment by cp2004** [comment]
All good

**Comment by The-EG** [comment]
Confirmed fixed in 1.7.0rc3
